test/e2e_tests/specs/ClearConversationContent/clearConversationContent.spec.ts

```diff
@@ -0,0 +1,300 @@
+        await userAPages.conversationList().openContextMenu(conversationName);
+        await userAPages.conversationList().clearContentButton.click();
+
+        // Step 4: Warning Popup should open and User A clicks 'Clear'
+        if (conversationType === 'group') {
+          await expect(userAModals.optionModal().modal).toBeVisible();
+          await userAModals.optionModal().clickAction();
+        } else {
+          await expect(userAModals.confirm().modal).toBeVisible();
+          await userAModals.confirm().clickAction();
+        }
+
+        // Step 5: Verify you can receive incoming new messages, pings, calls, pictures, and files
+        // 5.1 Messages
+        await userBPages.conversationList().openConversation(conversationPartnerForUserB);
+        await userBPages.conversation().sendMessage('Message from User B after Clear');
+        await userAPages.conversationList().openConversation(conversationName);
+        await expect(userAPages.conversation().messages).toHaveCount(1);
+
+        // 5.2 Pings
+        await userBPages.conversationList().openConversation(conversationPartnerForUserB);
+        await userBPages.conversation().sendPing();
+        await userAPages.conversationList().openConversation(conversationName);
+        await expect(userAPages.conversation().getPing()).toBeVisible();
+
+        // 5.3 Pictures
+        const {page} = userBPages.conversation();
+        await shareAssetHelper(getImageFilePath(), page, page.getByRole('button', {name: 'Add picture'}));
+
+        const messageWithImage = userAPages
+          .conversation()
+          .getMessage({sender: userB})
+          .filter({has: userAPages.conversation().page.locator('img[data-uie-name="image-asset-img"]')});
+        await expect(messageWithImage).toBeVisible();
+
+        // 5.4 Files
+        await userAPages.conversationList().openConversation(conversationName);
+        await shareAssetHelper(getTextFilePath(), page, page.getByRole('button', {name: 'Add file'}));
+        const messageWithFile = userAPages
+          .conversation()
+          .getMessage({sender: userB})
+          .filter({has: userAPages.conversation().page.locator('[data-uie-name="file-asset"]')});
+        await expect(messageWithFile).toBeVisible();
+
+        // 5.5 Calls
+        await userAPages.conversationList().openConversation(conversationName);
+        await userBPages.conversationList().openConversation(conversationPartnerForUserB);
+        await userBPages.conversation().startCall();
+        await expect(userAPages.calling().acceptCallButton).toBeVisible();
+      },
+    );
+  });
+
+  [
+    {tag: '@TC-424', type: 'clear', conversationType: 'group'},
+    {tag: '@TC-425', type: 'clear', conversationType: '1:1'},
+  ].forEach(({tag, type, conversationType}) => {
+    test(
+      `I want to ${type} the ${conversationType} conversation content from conversation details options`,
```

Comment:
Having identical parameters within a parameterized test makes them obsolete ;)

```suggestion
    {tag: '@TC-424', conversationType: 'group'},
    {tag: '@TC-425', conversationType: '1:1'},
  ].forEach(({tag, conversationType}) => {
    test(
      `I want to clear the ${conversationType} conversation content from conversation details options`,
```